seatunnel-formats/seatunnel-format-compatible-connect-json/src/main/java/org/apache/seatunnel/format/compatible/kafka/connect/json/CompatibleKafkaConnectDeserializationSchema.java

```diff
@@ -126,15 +127,13 @@ private SeaTunnelRow convertJsonNode(JsonNode jsonNode) {
         if (jsonNode.isNull()) {
             return null;
         }
+
         try {
             org.apache.seatunnel.shade.com.fasterxml.jackson.databind.JsonNode jsonData =
                     objectMapper.readTree(jsonNode.toString());
             return (SeaTunnelRow) runtimeConverter.convert(jsonData);
         } catch (Throwable t) {
-            throw new SeaTunnelJsonFormatException(
-                    CommonErrorCodeDeprecated.JSON_OPERATION_FAILED,
-                    String.format("Failed to deserialize JSON '%s'.", jsonNode),
-                    t);
+            throw mayThrow(t, () -> CommonError.jsonOperationError(FORMAT, jsonNode.toString(), t));
```

Comment:
I think just use `throw CommonError.jsonOperationError(FORMAT, jsonNode.toString(), t)` should be fine. Because it contains sub error information and `CommonError.jsonOperationError` information. So I think skip `CommonError.jsonOperationError` and only return sub error not a good choice. Same to all other use `mayThrow`, please do not use it for now.